submissions/adekemiAwomolo/dom/product.js

```diff
@@ -0,0 +1,61 @@
+var price = document.getElementById('price')
+quant = document.getElementById('product-quantity')
+btn = document.getElementById('order-btn');
+shipping = document.getElementById('shipping');
+total = document.getElementById('total');
+const Regex = /^[1-9]/;
+
+ function quantity_price(){
+     if(!quant.value.match(Regex)){
+         btn.disable = true;
+         btn.style.background = 'black';
+         quant.nextElementSibling.innerHTML ="  Kindly enter a valid Number";
+         quant.nextElementSibling.style.color = 'brown';
+        }
+        else{
+            btn.disable = false;
+            quant.nextElementSibling.innerHTML ="";
+            btn.style.background = 'orange';
+        }  
+        if(quant.value >= 1){
+           let productPrice = quant.value * 400;
```

Comment:
What does `400` represent?

Reply:
The price of the product ma

Reply:
Also, assign it to a variable

Reply:
Ok ma

Reply:
But it’s assigned to a variable already ma.